asyncapi/asb/listener.bal

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java as java;
+
+# Ballerina Azure Service Bus Message Listener.
+# Provides a listener to consume messages from the Azure Service Bus.
+@display {label: "ASB", iconPath: "resources/asb.svg"}
```

Comment:
Do we need to have another label here? 
`label: "ASB Trigger"` or `label: "ASB Listener"`

Reply:
We have followed the same structure as in https://github.com/ballerina-platform/asyncapi-triggers/blob/6d28aa11ac1a3c519e550f75e0bfd2bd03e1f773/asyncapi/slack/listener.bal#L4. I will recheck this again & do the update if necessary.

Reply:
Shall we rename this to Azure Service Bus?. WDYT? @SkNuwanTissera @chalitha1989 @indikasampath2000

Reply:
Yes +1 for `Azure Service Bus`